exercises/space-age/tests/space-age.rs

```diff
@@ -0,0 +1,81 @@
+// Students,
+//
+// These tests use the `num` crate to test that floating-point numbers are within an expected
+// range.
+//
+// The first time you run `cargo test`, the Num crate will automatically be downloaded
+// and installed. More information on crates can be found at
+// https://doc.rust-lang.org/book/guessing-game.html#generating-a-secret-number
+//
+extern crate space_age;
+extern crate num;
+
+use std::fmt::Display;
+use space_age::*;
+use num::{Float, NumCast};
+
+fn assert_in_delta<T: Float + Display>(expected: T, actual: T) {
```

Comment:
I suppose I'm not afraid to add a dependency if it's necesary (that's what cargo is for) but I want to hear the explanation for why it is necesary, rather than have `fn assert_in_delta(expected: f64, actual: f64)`. What does the bound of `Float` get us that we will not otherwise have? Does it allow students more flexibility in the return type of their `years_during` function?